### ISSUE TITLE
Config option for default chat color?

### ISSUE THREAD

**Issue by ryantheleach** [hsyyid/EssentialCmds#138]
What do you think of adding a config option for the default chat color used by /msg and chatting?

**Comment by ghost** [comment]
Yeah this is a great idea. Sometimes the default chat colors and /msg colors are hard to read or look odd.
